Source/WebCore/platform/mediastream/gstreamer/RealtimeOutgoingAudioSourceGStreamer.cpp

```diff
@@ -61,18 +61,20 @@ bool RealtimeOutgoingAudioSourceGStreamer::setPayloadType(const GRefPtr<GstCaps>
     // FIXME: We use only the first structure of the caps. This not be the right approach specially
     // we don't have a payloader or encoder for that format.
     GUniquePtr<GstStructure> structure(gst_structure_copy(gst_caps_get_structure(caps.get(), 0)));
-    const char* encodingName = gst_structure_get_string(structure.get(), "encoding-name");
-    if (!encodingName) {
+    auto encoding = String::fromLatin1(gst_structure_get_string(structure.get(), "encoding-name")).convertToASCIILowercase();
```

Comment:
You should be able to do the same thing, creating a lower case string from a StringView and you would save one string copy.